### ISSUE TITLE
UX:  Emaid input box to be autofocused

### ISSUE THREAD

**Issue by anu-007** [fossasia/open-event-webapp#1018]
in web app generator page the email box should be autofocused on loading the page.This will enhance the accessibility of the web app. 

@aayusharora if you think that is useful I would like to work on it.

**Comment by aayusharora** [comment]
Valid UX issue @anu-007 Go ahead !!

**Comment by aayusharora** [comment]
Thanks @anu-007. Solved
